### PR TITLE
Cancel memory leak test for ppc kvm interrupt type guest

### PR DESCRIPTION
**Pull request by maxujun**
ID:1573

Signed-off-by: Xujun Ma <xuma@redhat.com>

### CODE REVIEW

**Comment by maxujun** [comment]
@MiriamDeng @PaulYuuu Could you help review?

File: qemu/tests/block_check_memory_leak.py [hunk 1]
```diff
@@ -50,6 +51,11 @@ def _execute_io_in_guest():
 
     timeout = params.get_numeric('login_timeout', 360)
     session = vm.wait_for_login(timeout=timeout)
+    if arch.ARCH in ('ppc64', 'ppc64le'):
+        cmd_output = vm.monitor.info("pic", False)
+        if "irqchip: in-kernel" in cmd_output:
+            test.cancel("Skip test for xive kvm interrupt guest due to"
```
Comment (PaulYuuu) [inline qemu/tests/block_check_memory_leak.py:57]: Do we need to check if it's a XICS guest?

Comment (maxujun) [reply qemu/tests/block_check_memory_leak.py:58]: No,just need to check XIVE guest, have updated.

**Review by PaulYuuu — APPROVED** [review]

**Comment by MiriamDeng** [comment]
ACK

**Pull request merged**